ipaserver/install/server/__init__.py

```diff
@@ -548,6 +560,10 @@ def dm_password(self, value):
     def admin_password(self, value):
         validate_admin_password(value)
 
+    # always run sidgen task and do not allow adding agents on first master
+    add_sids = True
```

Comment:
AFAICT `add_sids` is only relevant when upgrading from an old server which did not have the `sidgen` plugin enabled, so it in fact should always be `False` on first master.